clock/scheduledJobs/jobCleanup.js

```diff
@@ -47,9 +47,10 @@ function execute(batchSize, delay) {
     if (n === 0) return Promise.resolve();
     const from = skippedJobCount;
     const to = skippedJobCount + n - 1;
+    if (!now) now = Date.now();
```

Comment:
style guide violation - even simple if blocks require curly braces

Reply:
This says it's ok: https://github.com/airbnb/javascript#blocks--braces
Those are the rules we're using, right?

Reply:
I stand corrected! Thanks for pointing that out.